src/test/scala/org/bitcoins/core/util/Base58Test.scala

```diff
@@ -53,5 +90,53 @@ class Base58Test extends FlatSpec with MustMatchers {
     Base58.encode(Base58.decode(multi)) must be (bitcoinj)
   }
 
+  it must "read base58_keys_valid.json and validate each case" in {
+    import org.bitcoins.core.util.testprotocol.Base58ValidTestCaseProtocol._
+    val source = Source.fromURL(this.getClass.getResource("/base58_keys_valid.json"))
+    val lines = try source.getLines.filterNot(_.isEmpty).map(_.trim) mkString "\n" finally source.close()
+    val json = lines.parseJson
+    val testCases : Seq[Base58ValidTestCase] = json.convertTo[Seq[Base58ValidTestCase]]
+    for {
+      testCase <- testCases
+    } yield {
+      testCase must be (Base58ValidTestCaseImpl(testCase.addressOrWIFPrivKey, testCase.hashOrPrivKey, testCase.configParams))
+      if (testCase.addressOrWIFPrivKey.isLeft) {
+        Base58.isValid(testCase.addressOrWIFPrivKey.left.get.value) must be (true)
+      }
+      else println("Base58.isValid(testCase.addressOrWIFPrivKey.right.get): " + Base58.isValid(testCase.addressOrWIFPrivKey.right.get))
```

Comment:
Is this even testing anything? Shouldn't there be a `must be` some where here to test the `right` value?